docs/specs/mapping.md

```diff
@@ -10,64 +10,58 @@ scopes of device data:
-  end
-  deactivate Agent
-  Note over Agent, Mapper: Mapping Start
-  activate Mapper
-  Agent->>Mapper: MAPPING CONFIG
-  Mapper->>Agent: MAPPING STATE
-  loop Devices
-    Mapper->>Agent: MAPPING EVENT<br/>(*guid, scan_id, *device_id)<br/><translations>
-    Agent->>Mapper: MAPPING COMMAND<br/>(device_id, *device_num_id)
-    Agent-->>Pipeline: Onboard RPC<br/>(guid, device_id, device_num_id)<br/><translations>
-  end
-  deactivate Mapper
-  Devices->>Pipeline: POINTSET EVENT<br/>(device_id, device_num_id)<br/><pointset>
+  participant Provisioning Engine
+  participant Registry as Registry<br/>(w/ Internal DB)
+  participant Mapping Service
+  participant Source Repo 
+  participant Modeling Phase
+  Devices->>Provisioning Engine: Incemental Results<br/>(Discovery Events)
+  Provisioning Engine->>Registry: Incremental Results
+  Devices->>Mapping Service: Discovery Complete Event
+  Source Repo->>Mapping Service: Base Model Import
+  Registry->>Mapping Service: All Results
+  Note over Mapping Service: Map Results
+  Mapping Service->>Source Repo: 'discovery' branch
+  Source Repo-->>Modeling Phase: Notify branch updated
 ```
 
-1. *(Fieldbus Discovery)* scan for fieldbus _device_ information from devices (e.g. BACnet, format out of scope for UDMI):
-  * "I am device `78F936` with points { `room_temp`, `step_size`, and `operation_count` }"
-2. **[Discovery Events](../../tests/schemas/events_discovery/enumeration.json)** wraps the device info from the discovery
-   into a UDMI-normalized format, e.g.:
-  * "Device `78F936` has points { }, with a public key `XYZZYZ`"
-3. **[Mapping Config](../../tests/schemas/config_mapping/mapping.json)** from the _agent_ indicates that the _mapper_ should export responses.
-3. **[Mapping Events](../../tests/schemas/events_mapping/mapping.json)** from the _mapper_ contain actual calculated point mappings:
-  * "Device `78F936` is an `AHU` called `AHU-183`, and `room_temp` is really a `flow_temperatue`"
-3. **[Mapping Command](../../tests/schemas/commands_mapping/mapping.json)** to the _mapper_ contain results of initial provisioning:
-  * "Device `78F936` has a numerical id `2198372198752`
-4. *(Onboard Info)* are sent to the _pipeline_ to onboard a device (contents are defined by _pipeline_ and out of scope for UDMI).
-8. **[Telemetry Events](../../tests/schemas/events_pointset/example.json)** are data events from _device_ to _pipeline_... business as usual:
-  * "I am `AHU-183`, and my `room_temp` is `73`"
+
+* **[Discovery Events](../../tests/schemas/events_discovery/enumeration.json)** wraps the device info from the discovery
+  into a UDMI-normalized format, e.g.:
+  "Device `78F936` has points { }, with a public key `XYZZYZ`"
+* **[Discovery Complete Event](../../validator/sequences/scan_single_future/events_discovery.json)** having `event_no` as negative value, with related [attributes](../../validator/sequences/scan_single_now/events_discovery.attr) 
```

Comment:
I'm not sure adding the `attributes` links helps anything -- it also begs why nothing else has it (e.g. discovery events have attributes as well)